### PR TITLE
Update .gitignore3

### PR DESCRIPTION
**Pull request by perfectacle2**
zxvxcv

### CODE REVIEW

**Review by perfectacle — COMMENTED** [review]
zxcv

**Review by perfectacle — APPROVED** [review]
zxcvwew

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@ build
 .idea
 .gradle
 gradlew
-gradlew.bat
\ No newline at end of file
+gradlew.bat
```
Comment (perfectacle) [inline .gitignore:5]: zxcv

**Review by perfectacle — CHANGES_REQUESTED** [review]
vzxcv

**Comment by perfectacle2** [comment]
ㅋㅌㅊㅍ

**Pull request closed without merging**